build-info-extractor/src/main/java/org/jfrog/build/extractor/scan/License.java

```diff
@@ -18,17 +18,24 @@
     private String fullName;
     private String name;
     private List<String> moreInfoUrl = new ArrayList<>();
+    private boolean isViolate;
```

Comment:
```suggestion
    private boolean violate;
```
Let's change 
`isViolate`, `getIsViolate` and `setIsViolate` 
to 
`violate`, `isViolate` and `setViolate`, to comply to Java standards:
https://en.wikibooks.org/wiki/Java_Programming/JavaBeans#Implementation